.github/workflows/sourcemaps.yml

```diff
@@ -0,0 +1,12 @@
+name: Source Maps 4 Bugsnag
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  upload_source_map:
+    runs-on: ubuntu-latest
+    name: Upload Source Maps to Bugsnag
+    run: npx bugsnag-source-maps upload-node --api-key bd22af1f4e1213b5badb590a48128bf1 --directory=./.next/static/chunks/
```

Comment:
I believe this is a more sensitive api key than the one given to the web client. Can you move it to a GitHub secret and rotate?

Reply:
funnily enough its the same one.